README.md

```diff
@@ -49,6 +49,10 @@ The port on which Solr will run.
 
 Memory settings for the JVM. These should be set as high as you can allow for best performance and to reduce the chance of Solr restarting itself due to OOM situations.
 
+    solr_timezone: "UTC"
+
+Solr used timezone
```

Comment:
For the documentation, please write something more descriptive? I can't find any documentation on `SOLR_TIMEZONE` configuration in the official docs, so it would be important to know why someone would want to customize it, more documentation on what it affects, etc. (Also the format).